packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {mount} from 'enzyme'
+
+import TimeEdit from './TimeEdit'
+
+const EMPTY_FUNC = () => {}
+
+describe('tocco-ui', () => {
+  describe('EditableValue', () => {
+    describe('typeEditors', () => {
+      describe('TimeEdit ', () => {
+        const valueObject = {
+          value: {hoursOfDay: 8, minutesOfHour: 45}}
+
+        test('should render DurationEdit', () => {
+          const wrapper = mount(
+            <TimeEdit value={valueObject} onChange={EMPTY_FUNC}/>
+          )
+          expect(wrapper.find('DurationEdit')).to.have.length(1)
```

Comment:
you could check if the output contains an "8" and the number "45". this would be a more or less useful unit test

Reply:
IMO this kind of test belongs to DurationEdit, as there already is.

Reply:
if you do white box tests you are right.